dyno-core/src/main/java/com/netflix/dyno/connectionpool/impl/ConnectionPoolConfigurationImpl.java

```diff
@@ -87,6 +87,29 @@ public ConnectionPoolConfigurationImpl(String name) {
 		this.localRack = ConfigUtils.getLocalZone();
 		this.localDataCenter = ConfigUtils.getDataCenter();
 	}
+
+	public ConnectionPoolConfigurationImpl(ConnectionPoolConfigurationImpl config) {
```

Comment:
The shadow pool needs its own configuration object. Rather than making ConnectionPoolConfigurationImpl ```Cloneable``` I added a copy constructor. 

I will add a javadoc comment to the constructor - thanks